platform/android/CHANGELOG.md

```diff
@@ -3,14 +3,34 @@
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
 ## master
+
+## 8.0.0-beta.1 - May 16, 2019
+
+### Features
+ - Option to provide custom values for location camera transition [#14606](https://github.com/mapbox/mapbox-gl-native/pull/14606)
+ 
+### Bugs
+ - Expose trackingGesturesManagement attribute [#14659](https://github.com/mapbox/mapbox-gl-native/pull/14659)
+ - Harden onLowMemory call, use atomic boolean for thread safe management of the render surface [#14622](https://github.com/mapbox/mapbox-gl-native/pull/14622) 
+ - Normalize previous rotation values of the location animator [#14626](https://github.com/mapbox/mapbox-gl-native/pull/14626)
+ - Remove missing image log [#14600](https://github.com/mapbox/mapbox-gl-native/pull/14600)
```

Comment:
This actually was released with `mojito`.

Reply:
weird that it's not showing in the commit itself https://github.com/mapbox/mapbox-gl-native/commit/ea50c776cd16a95a84718fcdcec608c407977068, normally it should be included in the tag.

Reply:
I think you're looking for this commit https://github.com/mapbox/mapbox-gl-native/commit/82206fb1fb6d6f06eea90d9717b2fad230ba5e12. Isn't tag only displayed on the actually tagged commit?

Reply:
:man_facepalming: